article/test/AnalyticsFeatureTest.scala

```diff
@@ -20,26 +20,7 @@ import conf.Configuration
 
     // Scenarios
 
-    scenario("Omniture tracks user actions") {
-
-      Given("I am on an article entitled 'Olympic opening ceremony will recreate countryside with real animals'")
-      goTo("/sport/2012/jun/12/london-2012-olympic-opening-ceremony") { browser =>
-        import browser._
-
-        Then("the Omniture webbug should record my visit")
-        val webbug = findFirst("#omnitureNoScript img")
```

Comment:
@mkopka @dominickendrick Hi both - this doesn't actually feel quite right to me - we don't have any `noscript` alternative to the omniture hits-secure pixel gif for GA. 

I would imagine that GA is therefore undercounting the PVs? 

This may have been a conscious decision, in which case I can leave the removal of the pageview pixel as is, otherwise we can repurpose it for GA.

Ps: [GA do provide the mechanism with utm.gif](https://developers.google.com/analytics/resources/concepts/gaConceptsTrackingOverview#gifParameters)